app/src/main/java/org/p2p/wallet/auth/interactor/UsernameInteractor.kt

```diff
@@ -41,7 +41,7 @@ class UsernameInteractor(
 //        }
     }
 
-    fun usernameExists(): Boolean = sharedPreferences.contains(KEY_USERNAME)
+    fun isUsernameExists(): Boolean = sharedPreferences.contains(KEY_USERNAME)
```

Comment:
that's not correct, `is` cannot be used with exist`s` endings.
Alternatives:  `isUsernameNotEmpty`